### PR TITLE
fix: Fixed docstrings for wildcards and matching engine type

### PR DESCRIPTION
**Pull request by ivanmkc**
Fixed incorrect docstrings.

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-aiplatform/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

Fixes: https://github.com/googleapis/python-aiplatform/issues/957

### CODE REVIEW

File: google/cloud/aiplatform/featurestore/entity_type.py [hunk 1]
```diff
@@ -1101,10 +1101,9 @@ def ingest_from_gcs(
                     higher than millisecond precision.
             gcs_source_uris (Union[str, List[str]]):
                 Required. Google Cloud Storage URI(-s) to the
-                input file(s). May contain wildcards. For more
-                information on wildcards, see
-                https://cloud.google.com/storage/docs/gsutil/addlhelp/WildcardNames.
-                Example:
+                input file(s).
```
Comment (ivanmkc) [inline google/cloud/aiplatform/featurestore/entity_type.py:1104]: None of the Dataset support wildcards. @nayaknishant are we sure that FeatureStore supports wildcards?

File: google/cloud/aiplatform/models.py [hunk 2]
```diff
@@ -2308,8 +2308,6 @@ def batch_predict(
             gcs_source: Optional[Sequence[str]] = None
```
Comment (ivanmkc) [inline google/cloud/aiplatform/models.py:2308]: Unsure if batch predict supports wildcards or not.

File: google/cloud/aiplatform/featurestore/entity_type.py (hunk 1 above)
Comment (nayaknishant) [reply google/cloud/aiplatform/featurestore/entity_type.py:1104]: hey Ivan, I'll take a look. I don't think so though.

Comment (ivanmkc) [reply google/cloud/aiplatform/featurestore/entity_type.py:1104]: Thanks!

Comment (nayaknishant) [reply google/cloud/aiplatform/featurestore/entity_type.py:1104]: Tested on https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/feature_store/sdk-feature-store.ipynb and seems to be working. We can keep the wildcards description for now.

File: google/cloud/aiplatform/models.py (hunk 2 above)
Comment (nayaknishant) [reply google/cloud/aiplatform/models.py:2307]: Tested on https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/automl/sdk_automl_video_action_recognition_batch.ipynb and does not support wildcards. Can delete the wildcards description.

**Review by nayaknishant — APPROVED** [review]

**Pull request merged**